[stable/kubewatch] Add more resources for watch

oleg-filiutsich opened the pull request:
<!--
Thank you for contributing to helm/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/helm/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/helm/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the
history. This will make it easier to identify new changes. The PR will be squashed
anyways when it is merged. Thanks.

For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file.

Please make sure you test your changes before you push them. Once pushed, a CircleCI
will run across your changes and do some initial checks and linting. These checks run
very quickly. Please check the results. We would like these checks to pass before we
even continue reviewing your changes.
-->

#### What this PR does / why we need it:
Add more resources for wath
```
  ingress: false
  namespace: false
  secret: false
  configmap: false
```


#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

billimek committed:
bump redis dependency version (#18381)

Signed-off-by: Jeff Billimek <jeff@billimek.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

anandsinghkunwar committed:
[stable/prometheus-operator] Made service port configurable (#18392)

* Prometheus/Alertmanager service port configurable
* Indentation fixes of comments in the `values.yaml` file

Signed-off-by: Anand Singh Kunwar <anandkunwar95@gmail.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

oleg-filiutsich committed:
[stable/kubewatch] Add ingress rule in rbac

Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

oleg-filiutsich committed:
[stable/kubewatch] Add more resources for watch

  ingress: true
  namespace: true
  secret: true
  configmap: true

Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

oleg-filiutsich committed:
[stable/kubewatch] Bump version 0.8.12 -> 0.8.13

Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

oleg-filiutsich committed:
[stable/kubewatch] Add more resources in readme

Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

oleg-filiutsich committed:
[stable/kubewatch] Add more resources in readme

Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

oleg-filiutsich committed:
[stable/kubewatch] Resources watch change default

  ingress: false
  namespace: false
  secret: false
  configmap: false

Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
[stable/ghost] Release 8.0.1 updating components versions (#18445)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

kvaps committed:
[stable/nfs-client-provisioner] fix imagePullSecrets (#18442)

* fix imagePullSecrets for nfs-provisioner

Signed-off-by: kvaps <kvapss@gmail.com>

* bump chart version

Signed-off-by: kvaps <kvapss@gmail.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

asosso committed:
[stable/prometheus-operator] Fix installation dashboard and prometheus rules in k8s 1.16. (#18401)

Set max_kubernetes version to 9.9.9-9

Signed-off-by: Andrea Sosso <andrea@sosso.me>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
[stable/testlink] Release 6.3.6 updating components versions (#18455)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
[stable/drupal] Release 5.2.9 updating components versions (#18456)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
rabbitmq-ha: automount service account API token (#17987)

Signed-off-by: Thierno BARRY <thierno.barry@polyconseil.fr>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

rjasper-frohraum committed:
[stable/wordpress] Add extra configuration parameters (#18421)

* [stable/wordpress] Add extra configuration parameters

This adds the following parameters:
- extraEnv
- extraVolumeMounts
- extraVolumes

Signed-off-by: Rico Jasper <rico.jasper@frohraum.de>

* [stable/wordpress] Bump chart version to 7.6.0

Signed-off-by: Rico Jasper <rico.jasper@frohraum.de>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
[stable/kong] simplify Ingress generation (#18061)

* [stable/kong] limit non-proxy ingresses to one host

Modify ingress configuration for services other than the proxy to
support a single hostname only.

Update proxy ingress TLS comments to reflect the actual expected value.

Signed-off-by: Travis Raines <traines@konghq.com>

* [stable/kong] add URL generation helper

Add a helper define to generate an appropriate external URL based on the
service's ingress configuration.

Signed-off-by: Travis Raines <traines@konghq.com>

* [stable/kong] populate service URLs automatically

Populate service URL hint settings based on ingress configuration.

Signed-off-by: Travis Raines <traines@konghq.com>

* [stable/kong] simplify Ingress generation

Generate Ingresses using a simplified, single-hostname and single-secret
configuration. Proxy Ingress generation remains unchanged to continue
support for multi-hostname proxy Ingresses.

Signed-off-by: Travis Raines <traines@konghq.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

gmenuel committed:
[stable/ambassador] Support for LB status being propagated to ingress (#18394)

* [stable/ambassador] add app.kubernetes.io/component label for service

Signed-off-by: Gregoire Menuel <gregoire.menuel@gmail.com>

* [stable/ambassador] Add downwardAPI to the deployment

Ambassador need this in order to know its labels

Signed-off-by: Gregoire Menuel <gregoire.menuel@gmail.com>

* [stable/ambassador] bump version to 4.4.4

Signed-off-by: Gregoire Menuel <gregoire.menuel@gmail.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

01100010011001010110010101110000 committed:
[stable/nginx-ingress] Fix HPA deployment apiVersion (#18428)

* Fix HPA deployment apiVersion

Signed-off-by: Tyler Gregory <tdgregory@protonmail.com>

* Bump chart version

Signed-off-by: Tyler Gregory <tdgregory@protonmail.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

bergemalm committed:
[stable/jenkins] Fix: Disable direct connection option default (#18410)

* [stable/jenkins] Fix: Disable direct connection option default (for kubernetes plugin version >= 1.20.2)

Signed-off-by: Mikael Bergemalm <mbergemalm@gmail.com>

* Added note about jnlp-slave version if direct connection is going to be used

Signed-off-by: Mikael Bergemalm <mbergemalm@gmail.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

mauermann committed:
[stable/influxdb] Fix backups to GCS (#18274)

* [stable/influxdb] fix gcs backup destination

Signed-off-by: Holger Mauermann <holger@mauermann.org>
Signed-off-by: Holger Mauermann <holger.mauermann@kiwigrid.com>

* [stable/influxdb] make secret key field configurable

Signed-off-by: Holger Mauermann <holger@mauermann.org>
Signed-off-by: Holger Mauermann <holger.mauermann@kiwigrid.com>

* [stable/influxdb] bump chart version

Signed-off-by: Holger Mauermann <holger@mauermann.org>
Signed-off-by: Holger Mauermann <holger.mauermann@kiwigrid.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

daanemanz committed:
Fix indentation on authorize deployment (#18465)

* [stable/pomerium] Fix indentation on livenessProbe

* [stable/pomerium] fix indentation on livenessProbe

The indentation for `initialDelaySeconds` placed it under the `tcpSocket`
where it's a parameter of the `livenessProbe` itself.

Signed-off-by: Daniel van den Berg <dvandenberg@optoro.com>

* [stable/pomerium] Version bump to 4.0.3

Version bump to 4.0.3

Signed-off-by: Daniel van den Berg <dvandenberg@optoro.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

diegocn committed:
[incubator/schema-registry] add external service annotations (#18352)

Signed-off-by: Diego Noguês <diego.sure@gmail.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

cryptobioz committed:
add support of bearer token for Prometheus' ServiceMonitor (#18413)

Signed-off-by: Léo Depriester <leo.depriester@camptocamp.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
[stable/phabricator] Release 8.0.3 updating components versions (#18470)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

winromulus committed:
[stable/mssql-linux] apiVersion compatibility (#18469)

* [stable/mssql-linux] apiVersion compatibility

Added deployment apiVersion compatibility switch based on k8s version.


Signed-off-by: Romeo Dumitrescu <winromulus@winromulus.com>

* Updated maintainers

Added myself as a maintainer and approver.

Signed-off-by: Romeo Dumitrescu <winromulus@winromulus.com>

* Fixed Chart.yaml missing a new line at the end


Signed-off-by: Romeo Dumitrescu <winromulus@winromulus.com>

* add new line

Signed-off-by: Maor Friedman <maor.friedman@redhat.com>

* remove new line

Signed-off-by: Maor Friedman <maor.friedman@redhat.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

winjer committed:
[stable/datadog] allow host networking for metrics (#15206)

* [stable/datadog] allow host networking for metrics

Signed-off-by: Doug Winter <doug.winter@isotoma.com>

* doc update

Signed-off-by: Doug Winter <doug.winter@isotoma.com>

* Update stable/datadog/templates/agent-service-metrics.yaml

Co-Authored-By: Cedric Lamoriniere <cedric.lamoriniere@datadoghq.com>
Signed-off-by: Doug Winter <doug.winter@isotoma.com>

* Update stable/datadog/values.yaml

Co-Authored-By: Cedric Lamoriniere <cedric.lamoriniere@datadoghq.com>
Signed-off-by: Doug Winter <doug.winter@isotoma.com>

* annotations is unused

Signed-off-by: Doug Winter <doug.winter@isotoma.com>

* Rework structure of values to match usage

Signed-off-by: Andrew Plummer <plummer574@gmail.com>

* Bump version

Signed-off-by: Andrew Plummer <plummer574@gmail.com>

* CR: remove docstring line

Signed-off-by: Andrew Plummer <plummer574@gmail.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
[stable/ghost] Release 8.0.2 updating components versions (#18471)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
[stable/osclass] Release 6.2.7 updating components versions (#18454)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

C123R committed:
feat:add support for providing clamd config (#18272)

Signed-off-by: cizer.ciz@gmail.com <cizer.ciz@gmail.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

juan131 commented on stable/kubewatch/README.md:
```diff
@@ -87,6 +87,11 @@ The following table lists the configurable parameters of the kubewatch chart and
 | `resourcesToWatch.job`                   | watch changes to [Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/) | `false`                           |
 | `resourcesToWatch.persistentvolume`      | watch changes to [PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)      | `false`                           |
+| `resourcesToWatch.ingress`      | watch changes to [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)      | `false`                           |
+| `resourcesToWatch.namespace`      | watch changes to [Namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)      | `false`                           |
```
What's the difference between this parameter and the parameter `namespaceToWatch`?

oleg-filiutsich replied:
Hi @juan131 .
**namespaceToWatch** configure kubewatch monitoring resources (deployment,pod and etc.) only in specific namespace (as space for resources).
**resourcesToWatch.namespace** tell kubewatch catch changes in create/reconfigure namespaces (as object) in all cluster

juan131 commented on stable/kubewatch/values.yaml:
```diff
@@ -45,6 +45,10 @@ resourcesToWatch:
+  ingress: false
+  namespace: false
+  secret: false
+  configmap: false
```
Please update the `values-production.yaml` too

oleg-filiutsich replied:
done

oleg-filiutsich replied:
done

juan131 commented on stable/kubewatch/Chart.yaml:
```diff
@@ -1,5 +1,5 @@
 name: kubewatch
-version: 0.8.12
+version: 0.8.13
```
Please bump minor version instead.

oleg-filiutsich replied:
please check again

juan131 reviewed (changes requested):
Hi @oleg-filiutsich 

Thanks so much for your PR! Please take a look to my suggestion when you have a chance.

vsliouniaev committed:
[stable/prometheus-node-exporter] Add vsliouniaev as owner (#18461)

* Add vsliouniaev as owner

Signed-off-by: vas <vasily.sliouniaev@jet.com>

* lint

Signed-off-by: vas <vasily.sliouniaev@jet.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

tembleking committed:
[stable/sysdig] Add 'How to upgrade to last version' to the README (#18403)

Signed-off-by: Federico Barcelona <fede_rico_94@hotmail.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
[stable/prometheus-operator] Add myself as owner/maintainer (#18424)

* Add myself as owner/maintainer

Signed-off-by: Bismarck Paliz <bismarck.paliz@jet.com>

* Changing ordering of maintainers/approvers/reviewers

Signed-off-by: Bismarck Paliz <bismarck.paliz@jet.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

alemorcuq committed:
Add LDAP support to RabbitMQ (#18488)

Signed-off-by: Alejandro Moreno <amoreno@bitnami.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

caarlos0 committed:
[stable/elasticsearch-exporter]: improved healthchecks (#16808)

* [stable/elasticsearch-exporter]: improved healthchecks

Signed-off-by: Carlos Alexandro Becker <caarlos0@gmail.com>

* fix: chart version

Signed-off-by: Carlos Alexandro Becker <caarlos0@gmail.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
[stable/minio] Fix pod annotations format in statefulset (#18487)

Signed-off-by: Tomas Makara <tomas.makara@gooddata.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
[stable/airflow] enabled ingress paths to be input through the values.yaml file (#18466)

* Enabled ingress paths to be input through the values.yaml file

Signed-off-by: Michael Hewitt <hmike1996@gmail.com>

* Bump up chart version

Signed-off-by: Michael Hewitt <hmike1996@gmail.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
[stable/grafana] Create all configmaps in range (#18247)

Move `---` inside the `range` block. It was previously outside
the block, so only one or zero configmaps would be made for
`.Values.dashboards`.

Signed-off-by: Sheldon Warkentin <sheldonw@forallsecure.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

liyinan926 committed:
[incubator/sparkoperator] Added missing RBAC permission for ResourceQuotas (#18501)

Signed-off-by: Yinan Li <ynli@google.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

mgrachev committed:
[bitnami/postgresql] Fix error when running init-container (#18447)

Error: failed to create containerd task: OCI runtime create failed: container_linux.go:337: starting container process caused "exec: \"sh\": executable file not found in $PATH": unknown
Signed-off-by: Mikhail Grachev <work@mgrachev.com>
Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
[stable/kubewatch] Refactoring after code review

Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>

Commit:
[stable/kubewatch] Add rbac for secrets and configmaps

Signed-off-by: Oleg Filychic <oleg.filychic@stark.games>
